src/main/resources/javascript/getAllElementsByPath.js

```diff
@@ -281,6 +281,32 @@ function isDisabled(node) {
     return node.disabled ? true : false;
 }
 
+// check if element is behind another one
+function isCovered(node) {
+    // TODO Handle false negatives for elements outside of viewport
+    if (typeof node.getBoundingClientRect === "function") {
+	    var boundingRect = node.getBoundingClientRect();
+
+	    var boundingLeft = boundingRect.left + 2;
+	    var boundingRight = boundingRect.right - 2;
+	    var boundingTop = boundingRect.top + 2;
+	    var boundingBottom = boundingRect.bottom - 2;
+	
+	    var topLeft = document.elementFromPoint(boundingLeft, boundingTop);
+	    var topRight = document.elementFromPoint(boundingRight, boundingTop);
+	    var bottomLeft = document.elementFromPoint(boundingLeft, boundingBottom);
+	    var bottomRight = document.elementFromPoint(boundingRight, boundingBottom);
+	    if ((topLeft != null && !node.contains(topLeft))
+	    	|| (topRight != null && !node.contains(topRight))
+	    	|| (bottomLeft != null && !node.contains(bottomLeft)) 
+	    	|| (bottomRight != null && !node.contains(bottomRight))) {
+	        return true;
```

Comment:
@diba1013 do we use a specific JS formatter here? Looks a bit odd to me, and it seems like we are mixing tabs and whitespaces. I'd rather do:

```suggestion
	    if ((topLeft != null && !node.contains(topLeft))
	    	    || (topRight != null && !node.contains(topRight))
	    	    || (bottomLeft != null && !node.contains(bottomLeft)) 
	    	    || (bottomRight != null && !node.contains(bottomRight))) {
	        return true;
```

Reply:
I personally would have just used the vscode formatter, and we do have one for [recheck-web-js](https://github.com/retest/recheck-web-js/blob/master/.prettierrc.js) using [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). But I am afraid, that we did not setup any proper rules for that. 

Personally, I have gone with a similar to our existing java formatter, using a mix of tsfmt and tslint (soon to be replaced with eslint) to ensure some formatting rules in my typescript project.

Reply:
OK, since we hopefully soon integrate recheck-web-js, I mark this as resolved. We can have a separate clean-up PR.